core/pkg/evaluator/fractional.go

```diff
@@ -27,7 +27,7 @@ func NewFractional(logger *logger.Logger) *Fractional {
 func (fe *Fractional) Evaluate(values, data any) any {
 	valueToDistribute, feDistributions, err := parseFractionalEvaluationData(values, data)
 	if err != nil {
-		fe.Logger.Error(fmt.Sprintf("parse fractional evaluation data: %v", err))
+		fe.Logger.Warn(fmt.Sprintf("parse fractional evaluation data: %v", err))
```

Comment:
Note - noted this while local validation. Error logs add stack traces, which is an overkill. Hence reduced to warn level.